recipes/ponio/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cmake ${CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX=$PREFIX $SRC_DIR -DCMAKE_INSTALL_LIBDIR=lib  -B build
```

Comment:
All part of $CMAKE_ARGS
```suggestion
cmake ${CMAKE_ARGS}  $SRC_DIR  -B build
```